assets/src/domain/eventEditor/ui/styles.scss

```diff
@@ -4,6 +4,8 @@
 }
 
 .post-type-espresso_events {
+	overflow-y: overlay;
```

Comment:
this closes this: https://github.com/eventespresso/event-espresso-core/issues/2867

Reply:
![Peek 2020-05-26 17-21](https://user-images.githubusercontent.com/1477453/82912177-6f913a00-9f75-11ea-8627-6d54a983a9b3.gif)

Reply:
**NOICE** 💯 👌